process/nodeStatusProcessor.go

```diff
@@ -451,21 +451,21 @@ func (nsp *NodeStatusProcessor) GetGenesisNodesPubKeys() (*data.GenericAPIRespon
 		return nil, err
 	}
 
+	var response data.GenericAPIResponse
```

Comment:
inconsistency in the PR: in some places, the response is defined as pointer and in other places, no

Reply:
also, in some places they are declared as `var response VarType` and in other places `response := VarType`

Reply:
fixed